hello-world.json

```diff
@@ -0,0 +1,33 @@
+{
+   "#": [
+     "Language implementations vary on the issue of missing input variables.",
+     "In this set of test cases, the input variable is left out of the definition,",
+     "and the generator would need to handle the language implementation."
+   ],
+   "cases": [
+      {
+         "description": "no name",
+         "expected": "Hello, World!",
+         "msg": ["When given no name, it should greet the world!"]
```

Comment:
That was the weird case with an empty name, right? I think we should remove that one.